exercises/acronym/src/example.c

```diff
@@ -6,7 +6,8 @@ static int is_first_letter_of_word(int character, int preceding_character)
 {
    return isalpha(character)
        && ((preceding_character == ' ')
-           || (preceding_character == '-')) ? 1 : 0;
+           || (preceding_character == '-')
+           || (preceding_character == '_')) ? 1 : 0;
```

Comment:
Is the `? 1 : 0` actually necessary?

Reply:
I'm fairly certain the answer is no. We should probably tweak this to return a `bool` for clarity.